### PR TITLE
Add lucasmoura as contributor

### PR DESCRIPTION
**Pull request by lucasmoura**
Add lucasmoura as contributor in the project

### CODE REVIEW

File: tools/.github-cla-signers [hunk 1]
```diff
@@ -2,3 +2,4 @@ beezly
 dhensby
 nishigori
 TheRealFalcon
+lucasmoura
```
Comment (OddBloke) [inline tools/.github-cla-signers:5]: Let's keep this list sorted. Perhaps we also need a doc update to instruct people to do that?

Comment (lucasmoura) [reply tools/.github-cla-signers:5]: I can update the documentation as well. Is there a good place to put that information ?

Comment (lucasmoura) [reply tools/.github-cla-signers:5]: Also, for this particular instance, should I do an amend to the original commit or create new a one sorting the list ?

Comment (OddBloke) [reply tools/.github-cla-signers:5]: Actually, I think we're missing docs for this ATM, so there aren't any to update. I've been meaning to add this, and briefly forgot that I hadn't yet. Apologies!

Comment (OddBloke) [reply tools/.github-cla-signers:5]: > Also, for this particular instance, should I do an amend to the original commit or create new a one sorting the list ?

Good question! We squash-merge everything in cloud-init (and curtin, for that matter), so either will result in exactly the same commit on master. :-)

Comment (lucasmoura) [reply tools/.github-cla-signers]: @OddBloke Done, I have sorted the list. Also, if you want I can work on that documentation

Comment (OddBloke) [reply tools/.github-cla-signers]: I think the change is going to require reworking/reorganising some existing docs, so it's probably not a great first week task, but thank you for the offer!

**Pull request merged**